src/lib.rs

```diff
@@ -115,7 +155,9 @@ impl Default for Collector {
     /// let collector = Collector::default();
     /// ```
     fn default() -> Self {
-        Self { prefix: "".into() }
+        Self {
+            metrics: Arc::default(),
+        }
```

Comment:
We can use `#[derive(Default)]` instead?

Reply:
Of course we can. In the previous version, `Default` was implemented manually
with provided doc comment. I'll change this to `derive` if it's ok.